x-pack/test/smb-fixture/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:16.04
+RUN apt-get update -qqy && apt-get install -qqy samba ldap-utils
```

Comment:
Now that `installsmb.sh` is being run in the docker file, I think installing samba and ldap-utils should move back into there?

Reply:
I don't have a strong opinion on that, and don't mind doing it in a follow up. 
One advantage of keeping these separate  is that one can iterate on the install script without waiting for apt all the time, but having everything in the same script is also nice.